akka-projection-alpakka-kafka/src/main/scala/akka/projection/scaladsl/kafka/KafkaConsumer.scala

```diff
@@ -0,0 +1,24 @@
+package akka.projection.scaladsl.kafka
+
+import akka.{Done, NotUsed}
+import akka.kafka.ConsumerMessage.{Committable, CommittableMessage}
+import akka.kafka.{CommitterSettings, ConsumerSettings, Subscription}
+import akka.kafka.scaladsl.{Committer, Consumer}
+import akka.kafka.scaladsl.Consumer.Control
+import akka.stream.scaladsl.{Flow, Source}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+// Either users need to build their own at-least-once Kafka source or we provide this convenience in Alpakka
+object KafkaConsumer {
+
+  def atLeastOnce[K, V](settings: ConsumerSettings[K, V],
+                        subscription: Subscription,
+                        committerSettings: CommitterSettings)(handler: CommittableMessage[K, V] => Future[_])(implicit ec: ExecutionContext): Source[Done, Control] = {
```

Comment:
What would be a use-case for a `Source[Done]`? (Other than in the projections API.)

Reply:
Actually, we are only interested in `Control`.

Reply:
I added my suggestion for this in https://github.com/akka/alpakka-kafka/pull/1090